docs/examples/Validation Examples.ipynb

```diff
@@ -195,11 +195,11 @@
      "output_type": "stream",
      "text": [
       "There are 5 validation errors in the data:\n",
-      "   1. Fields line.id and sym_power_sensor.id are not unique for 2 lines/sym_power_sensors.\n",
-      "   2. Field 'to_node' does not contain a valid node id for 1 line.\n",
-      "   3. Field 'power_sigma' is not greater than zero for 2 sym_power_sensors.\n",
-      "   4. Field 'measured_object' does not contain a valid line/transformer id for 1 sym_power_sensor. (measured_terminal_type=branch_from)\n",
-      "   5. Field 'measured_object' does not contain a valid source id for 1 sym_power_sensor. (measured_terminal_type=source)\n"
+      "   1. Fields ComponentType.line.id and ComponentType.sym_power_sensor.id are not unique for 2 ComponentsTypes.lines/ComponentsTypes.sym_power_sensors.\n",
+      "   2. Field 'to_node' does not contain a valid node id for 1 ComponentType.line.\n",
+      "   3. Field 'power_sigma' is not greater than zero for 2 ComponentsTypes.sym_power_sensors.\n",
+      "   4. Field 'measured_object' does not contain a valid line/generic_branch/transformer id for 1 ComponentType.sym_power_sensor. (measured_terminal_type=branch_from)\n",
+      "   5. Field 'measured_object' does not contain a valid source id for 1 ComponentType.sym_power_sensor. (measured_terminal_type=source)\n"
```

Comment:
I know you refer to components here in plural, but using `ComponentTypes` (with an `s`) may lead to confusion. The same with the names of the actual components, such as `lines`, `sym_power_sensors`, etc. It also happens in other places.

Reply:
I think you answered another question here (?)

Reply:
yes, apologies. i thought something went wrong when i clicked submit but your comment is correct indeed.